G1ANT.Addon.Net/API/SmtpManager.cs

```diff
@@ -22,23 +19,21 @@ public sealed class SmtpManager
         private void ConnectClient(SmtpClient client)
         {
             client.Connect(host, port, socketOptions);
-            if (credentials != null)
-                client.Authenticate(credentials);
+            authenticator?.Authenticate(client);
```

Comment:
Maybe we should throw exception when authenticator is null.

Reply:
it is possible to not pass authenticator to imap.openex/smtp.openex, this is to allow to connect to the server which doesn't need authentication